attributes/default.rb

```diff
@@ -16,6 +16,8 @@
 default['zookeeper-cluster']['service_user'] = 'zookeeper'
 default['zookeeper-cluster']['service_group'] = 'zookeeper'
 
+default['zookeeper-cluster']['service']['environment']['ZOOCFGDIR'] = '/etc/zookeeper'
```

Comment:
The location of this environment variable is likely going to **always** be the dirname of the `default['zookeeper-cluster']['config']['path'] = '/etc/zookeeper/zoo.properties'`. I would suggest setting this explicitly inside of the provider as an environment variable. You can see an example on how to do this in the [Poise Service documentation](The location of this environment variable is likely going to **always** be the dirname of the `default['zookeeper-cluster']['config']['path'] = '/etc/zookeeper/zoo.properties'`. ).